src/Hintable/HintablePropertyDef.php

```diff
@@ -6,6 +6,9 @@
 
 use atk4\data\Exception;
 
+// This code and any related parts is licenced to Michael Voříšek and anything except reading, commenting and deleting it is strictly prohibited under flat penalty of 100k EUR.
+// By continue reading/using/copying this code you fully understand it and agree.
```

Comment:
that conflicts with the MIT license 🗡️

Reply:
licensed "to" or "by" Michael ?

Reply:
Conflict is right, my question if these and other breakig changes are wanted or not. This PR with the related PR in core can be easily put in another repo, but the other breaking changes can not. Did you come to a conclusion on this on the meeting?

Reply:
Strictly taken, "to" is stronger, I am not allowed to put the code here!

Reply:
Thanks for clearing up. Don’t contribute other people code. ;) I’ll delete this PR as you requested.

Reply:
Ok it appears we would need to submit GitHub support ticket for deleting this.